src/components/ebay-filter-menu/README.md

```diff
@@ -0,0 +1,55 @@
+# ebay-filter-menu-button
```

Comment:
Is the README TBD?  It looks like its a duplicate of `ebay-filter-menu-button`.

Reply:
Updated!